core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -1010,7 +1011,7 @@ public void doListenerSweep() {
   public <T> void syncDocumentFromRemote(
```

Comment:
This class still needs to be updated to configure once per namespace and remove codecs and handlers/resolvers from these doc specific methods.

Reply:
👍